crates/extensions/tedge-p11-server/src/main.rs

```diff
@@ -150,8 +214,6 @@ async fn main() -> anyhow::Result<()> {
     let args = Args::parse();
 
     tracing_subscriber::fmt()
-        .with_file(true)
-        .with_line_number(true)
```

Comment:
I propose to remove these two lines as this adds noise for non dev users.

Reply:
I'd be for enabling it only when `RUST_LOG` is present. Should be easily achieved by replacing true with `std::env::var("RUST_LOG").is_ok()`

Reply:
Done: https://github.com/thin-edge/thin-edge.io/pull/3586/commits/789f202696b45c398f000ffc8f29a39966000674